src/components/UserInfo/UserInfo.js

```diff
@@ -41,12 +42,15 @@ export default function UserInfo() {
     if (user) {
       var username = user.get("username");
       var total_points = user.get("total_points");
+      var totalCoins = user.get("coins");
+      console.log(user.get("coins"));
```

Comment:
i guess once you're sure the code works, there's no need to add logging statements anymoer.